back-office/shop-parameters/order-settings/order-setting-statuses-page.md

```diff
@@ -1,48 +1,167 @@
+
+3) Icon 
+
+This should be removed during the migration since it is not used anymore.
+
+4) Send email to the customer
+
+The merchant can choose to send an email or not to the customer when the order status is applied to the order.
 
-The default status or the status added by modules can’t be deleted.
+A search can be made by selecting "yes" or "no" through the drop-down.
+
+5) Delivery
+
+The merchant can choose to show the delivery PDF or not when the order status is applied to the order.
+
+A search can be made by selecting "yes" or "no" through the drop-down.
+
+6) Invoice
+
+The merchant can choose to allow the customer to download and view PDF versions of his/her invoices or not when the order status is applied to the order.
+
+A search can be made by selecting "yes" or "no" through the drop-down.
+
+7) Email template
+
+It's possible to sort by email template asc or desc and search by email template. If no email template is selected, then it's empty.
+
+8) Actions
+
+There are two actions:
+
+- Edit, the main one:
+
+After clicking on it, the order status is opened in the same tab.
+This action can also be performed by clicking on the line.
+
+- Delete, the second one:
+
+This option isn't displayed for the default status and the status added by modules.
+
+After clicking on it, a modal is opened asking to confirm or cancel the action: "Are you sure you want to delete the selected item(s)?"
+
+The action can be canceled by clicking on the cross or on the cancel button.
+
+After clicking on "Delete" button, the message is deleted and a successful message is displayed "Successful deletion."
+
+#### Search
+
+Below, the same behavior as others listing:
+
+By default, the search button is disabled. It becomes enabled when something is filled in a search field.
+
+After clicking on "Search", the search is applied and the order statuses matching with the search are displayed
+
+If no results are found, then a message "No records found" is displayed.
+
+After clicking on "Search", this button is disabled and a "Reset" button is displayed.
+
+After clicking on "Reset", the search is removed and all order statuses are displayed.
+
+#### Pagination
+
+The pagination of the list starts when there are more than 20 order statuses.
+
+The merchant can select the number of order statuses displayed through a drop-down: 20, 50, 300, 1000
+
+When there are more results than the number selected, the merchant can navigate through pages.
+
+If a sort and/or a search have been done, results should be kept when moving on to the next page.
+
+### Add / edit order status
 
 When **adding a new order status or editing an order status**, the user can fulfill the information below:
 *   **Status name**, mandatory information
 *   **Icon**. (to be removed after migration since it is not used anymore),
-*   **Label color** with a color picker or write the hexadecimal code
+*   **Color** with a color picker or write the hexadecimal code
```

Comment:
What should happen when the color field is left blank ? (maybe it's out of scope for this PR)

Reply:
I added the default value of the field: #ffffff
There is an issue when the color is not defined: https://github.com/PrestaShop/PrestaShop/issues/21284
I linked the issue, the behavior should still be defined